admin/class-customizer.php

```diff
@@ -0,0 +1,243 @@
+<?php
+/**
+ * @package WPSEO\Admin\Customizer
+ */
+
+/**
+ * Class with functionality to support WP SEO settings in WordPress Customizer.
+ */
+class WPSEO_Customizer {
+
+	/**
+	 * Construct Method.
+	 */
+	function __construct() {
+		add_action( 'customize_register', array( &$this, 'wpseo_customize_register' ) );
+	}
+
+	/**
+	 * Function to support WordPress Customizer
+	 *
+	 * @param WP_Customize_Manager $wp_customize
+	 */
+	function wpseo_customize_register( $wp_customize ) {
```

Comment:
This is one beast of a function. There's a lot of repetition in the `add_control` and `add_setting` function calls within, so I'd build wrapper functions for each, then split into separate private functions for each setting+control combo, then fire those from here.